### ISSUE TITLE
Hi, I want to know what's the meaning of subdivisions 

### ISSUE THREAD

**Issue by GuoQuanhao** [Tianxiaomo/pytorch-YOLOv4#463]
```python
if global_step % config.subdivisions == 0:
    optimizer.step()
    scheduler.step()
    model.zero_grad()
```

**Comment by lixinghe1999** [comment]
From my understanding, it accumulate multiple times of gradient descent, so when your GPU can only afford 4 batchsize, you can simulate that it can afford 16(4*4). Of course, this trick will not make your training faster, it only may help it converge.

**Comment by GuoQuanhao** [comment]
Oh, thanks, I think you are right. I think it's amazing, I have never seen this.
